recommended.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+const prettier = require('./eslint-plugin-prettier');
+// eslint-disable-next-line n/no-unpublished-require
```

Comment:
It seems we're missing it from `peerDependencies`

Reply:
Yeah I think this is an error from before, it's included in `peerDependenciesMeta` but not in `peerDependencies` as it should. I can add it – but I suspect it can be seen as a breaking change to introduce a peer dependency, no?

Reply:
Added it here: https://github.com/prettier/eslint-plugin-prettier/pull/592/commits/35363bbe11a4cbed97ad679af778203a24d8c3a1

Reply:
It's a bugfix IMO, and we've declared it as optional already.

Reply:
I removed it from peer dependencies here: https://github.com/prettier/eslint-plugin-prettier/pull/592/commits/e90c10892614d5ed71395123f1a112a71d8760dc